test/parallel/test-blob.js

```diff
@@ -269,6 +269,29 @@ assert.throws(() => new Blob({}), {
   reader.closed.then(common.mustCall());
 })().then(common.mustCall());
 
+(async () => {
+  const b = new Blob(['A', 'B', 'C']);
+  const stream = b.stream();
+  const chunks = [];
+  const decoder = new TextDecoder();
+  await stream.pipeTo(new WritableStream({
+    write(chunk) {
+      chunks.push(decoder.decode(chunk, { stream: true }));
+    }
+  }));
+  assert.strictEqual(chunks.join(''), 'ABC');
+})().then(common.mustCall());
+
+(async () => {
+  const file = new Blob(['<svg></svg>'], { type: 'image/svg+xml' });
+  const url = URL.createObjectURL(file);
```

Comment:
This test fails on linux built without crypto so need to add a check for that could you add it @bellbind or i could too

Reply:
Just need to add this 
```js
if (!common.hasCrypto)
  common.skip('node compiled without crypto.');
```